opentelemetry-user-events-logs/src/logs/reentrant_logprocessor.rs

```diff
@@ -26,6 +26,8 @@ impl<T: LogExporter> ReentrantLogProcessor<T> {
 impl<T: LogExporter> opentelemetry_sdk::logs::LogProcessor for ReentrantLogProcessor<T> {
     fn emit(&self, record: &mut SdkLogRecord, scope: &InstrumentationScope) {
         let log_tuple = &[(record as &SdkLogRecord, scope)];
+        // TODO: Using futures_executor::block_on can make the code non reentrant safe
```

Comment:
same risk for EventHeader library, but that library is meant for low level interaction with Linux tracepoints, so unlikely to introduce logging of its own.